### DISCUSSION TITLE
Can't turn keepInventory to false (need help!)

### DISCUSSION THREAD

**Discussion by aokichangaming** [EssentialsX/Essentials#4186] [Q&A] [answered]
**/ess version output:**
[INFO] Server version: 1.12.2-R0.1-SNAPSHOT git-Spigot-dcd1643-e60fc34 (MC: 1.12.2)
[INFO] EssentialsX version: 2.17.1.0
[INFO] LuckPerms version: 5.1.26
[INFO] Vault version: 1.7.2-b107
[INFO] EssentialsXChat version: 2.17.1.0

**Server startup log:**
https://gist.github.com/aokichangaming/51b22279c13aff5f4f23607ae236e56c

**EssentialsX config files:**
https://gist.github.com/aokichangaming/6e0402c20811c162bed78bb6cf5db79a

**Problem**

I can't turn keepInventory to false in any of my worlds. It continues to appear as though I have turned keepInventory to false, but I still keep my items when I die.

I've tried using /mv gamerule keepInventory false (multiverse-core plugin) & /gamerule keepInventory false in the worlds but it hasn't worked.

![Gamerules](https://user-images.githubusercontent.com/52972577/120255837-79b5da80-c252-11eb-9946-25e43158620f.JPG)

**Comment by benwoo1110** [comment]
Check you you didn’t give your players essentials.keepinv permission.

**Comment by JRoy** [comment, marked as answer]
You have to not give the `*` permissions or properly negate `essentials.keepinv`

**Reply by aokichangaming** [reply]
> You have to not give the `*` permissions or properly negate `essentials.keepinv`

Okay, I'll check that out right now.

**Reply by aokichangaming** [reply]
Thanks for your help. That worked.
